monai/losses/dice.py

```diff
@@ -147,7 +147,9 @@ def forward(self, input: torch.Tensor, target: torch.Tensor, smooth: float = 1e-
 
 class MaskedDiceLoss(DiceLoss):
     """
-    Same as DiceLoss, but accepts a binary mask ([0,1]) indicating a region over which to compute the dice.
+    Add an additional `masking` progress before `DiceLoss`, accept a binary mask ([0, 1]) indicating a region,
```

Comment:
"`masking` progress" -> "`masking` process" ?

Reply:
updated in the latest commit.
Thanks.